### ISSUE TITLE
Cannot see logs from slave on master

### ISSUE THREAD

**Issue by ray-matos-mpf** [jhuckaby/Cronicle#211]
## Summary

Added a slave to the master, i can access the slave via the internet, i cannot log into the slave with my admin user/pass.  Also on the master, for a job running on the slave, i do not see any logs, However the progress bar does work.  

## Your Setup

1 master, 1 slave, both have access to internet

### Operating system and version?

Ubuntu 18

### Node.js version?

10

### Cronicle software version?

 0.8.31

### Are you using a multi-server setup, or just a single server?

Master with 1 slave

### Are you using the filesystem as back-end storage, or S3/Couchbase?

no

**Comment by ray-matos-mpf** [comment]
Adding to this, its just live logs, when the job completes i see the full logs.

**Comment by jhuckaby** [comment]
Please see [Issue #206](https://github.com/jhuckaby/Cronicle/issues/206) -- it may be related.

**Comment by raymatos** [comment]
Its the same job that is running on the master where live logs work fine.   Also when the job completes i see the full log there.  So its just live logs not working.

**Comment by servo1x** [comment]
Having the same issue here. Were you able to resolve @raymatos ?

**Comment by raymatos** [comment]
Yes, i created a NFS share and mounted that on all the slaves, it now writes the logs to a shared location and I am now able to see them.

**Comment by servo1x** [comment]
@raymatos does this allow live logs to work? or is writing it to a share location, just an alternative for making it accessible for you?
